src/hatch_openzim/files_install.py

```diff
@@ -0,0 +1,199 @@
+import os
+import shutil
+import tempfile
+import zipfile
+from pathlib import Path
+from typing import Any, Dict, List
+from urllib.request import urlopen
+
+import toml  # to be replaced by tomllib once only 3.11 and above is supported
+
+from hatch_openzim.shared import log
+
+DEFAULT_OPENZIM_TOML_LOCATION = "openzim.toml"
+
+
+def process(openzim_toml_location: str = DEFAULT_OPENZIM_TOML_LOCATION):
+    """performs openZIM operations on files, i.e. download and extract ZIPs
+
+    Configuration is read from openzim.toml which must be in current directory
+
+    openzim_toml_location: location of the openzim.toml file with instructions about
+    files to install locally
+    """
+    openzim_file = Path(openzim_toml_location)
+    if not openzim_file.exists():
```

Comment:
A single `if` with two conditions seems more appropriate

Reply:
It means repeating the `not config_path.exists()` condition:

```
    if not config_path.exists() and openzim_toml_location != DEFAULT_OPENZIM_TOML_LOCATION:
        raise Exception(f"File is missing at {openzim_toml_location}")
    elif not config_path.exists():
        return
```

I find current code more clear / less error prone (even if it means more indentation, it is bearable). WDYT?

Reply:
You're right